src/main/resources/gscribe.sql

```diff
@@ -0,0 +1,134 @@
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `answers`
+--
+
+DROP TABLE IF EXISTS `answers`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `answers` (
+  `takenID` int NOT NULL,
+  `answer` json NOT NULL,
+  `Timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`takenID`),
+  CONSTRAINT `answersDB_to_examtakenDB_takenID` FOREIGN KEY (`takenID`) REFERENCES `examtaken` (`examTakenID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `exam`
+--
+
+DROP TABLE IF EXISTS `exam`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `exam` (
+  `examID` int NOT NULL AUTO_INCREMENT,
+  `createdBy` varchar(300) NOT NULL,
+  `spreadsheetId` varchar(100) NOT NULL,
+  `duration` int NOT NULL,
+  `createdOn` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`examID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `examtaken`
+--
+
+DROP TABLE IF EXISTS `examtaken`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `examtaken` (
+  `examTakenID` int NOT NULL,
+  `examID` int NOT NULL,
+  `studentEmailID` varchar(300) NOT NULL,
+  `studentRollNum` int NOT NULL,
+  `Timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`examTakenID`),
+  KEY `examtakenDB_to_examDB_examID_idx` (`examID`),
+  CONSTRAINT `examtakenDB_to_examDB_examID` FOREIGN KEY (`examID`) REFERENCES `exam` (`examID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `examtakenmetadata`
+--
+
+DROP TABLE IF EXISTS `examtakenmetadata`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `examtakenmetadata` (
+  `examTakenID` int NOT NULL,
+  `startTIme` timestamp NULL DEFAULT NULL,
+  `endTime` timestamp NULL DEFAULT NULL,
+  PRIMARY KEY (`examTakenID`),
+  CONSTRAINT `metadataDB_to_examtakenDB_examtakenID` FOREIGN KEY (`examTakenID`) REFERENCES `examtaken` (`examTakenID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `questions`
+--
+
+DROP TABLE IF EXISTS `questions`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `questions` (
+  `examID` int NOT NULL,
```

Comment:
exam_id

Reply:
Follow this convention for naming: https://medium.com/@centizennationwide/mysql-naming-conventions-e3a6f6219efe

Reply:
followed the name convention to change the database schema accordingly.